src/pages/guides/getting_started/code_playground.md

```diff
@@ -12,54 +14,223 @@ keywords:
+
+#### How to Use Add-on Mode:
+
+1. Click on the "Add-on" button on the right of the "Script" button in the top left corner of the playground window.
+2. Write code for your add-on in each of the supplied tabs (described below). This includes HTML, CSS, and JavaScript code that will run in the iframe UI and (optionally) call the Document APIs to interact directly with the Express document (optionally).
+3. Click "Run" to execute your script directly on the current document.
+
+  ![Code Playground Add-on Mode](./img/addon-mode.png)
+
+### HTML Tab
+
+This tab is for writing HTML code that defines the structure of your add-on's user interface. You can create elements like buttons, text fields, and layout containers here. Functionally, this tab mirrors the role of the `index.html` file you'd use in a typical add-on project.
+
+### CSS Tab
+
+Style your add-on's HTML elements in this tab. Create a visually appealing interface consistent with Adobe Express design patterns. This section corresponds to the `styles.css` file in a standard add-on project.
+
+### Iframe JS Tab
+
+This tab is for writing JavaScript code that runs in the iframe context of your add-on. Here, you can interact with:
+
+- The [Add-on UI SDK (`addOnUISdk`)](../../references/addonsdk/index.md)
+- The DOM elements in your HTML
+- Event handlers for your UI components
+
+This environment corresponds to the code you would typically write in your `index.js` or UI JavaScript files in a full add-on project.
+
+### Document JS Tab
+
+This tab is where you write JavaScript code that interacts directly with the Adobe Express document. It runs in the [Document Sandbox](../../references/document-sandbox/index.md) environment and gives you access to:
+
+- Document manipulation capabilities with the [Document APIs](../../references/document-sandbox/document-apis/index.md)
+- [Communication APIs](../../references/document-sandbox/communication/index.md) to facilitate interaction between the iframe context and the Document Sandbox.
+
+The Document JS tab corresponds to the code typically found in the `code.js` file of a complete add-on project.
+
+### Transitioning from Script Mode to Add-on Mode
+
+Once you've developed your code in Script mode, you can easily transition to Add-on mode to build a user interface around your functionality. Here's how:
+
+1. Use the "Copy" button in the top right corner of the playground while in Script mode to quickly copy your code to the clipboard.
+2. Click the "Add-on" button next to the "Script" button in the top left corner of the playground.
+3. Paste the code into the "Document JS" tab.
+4. Modify your script code as needed to be used in the add-on context along with your front-end logic in the HTML, Iframe, and CSS tabs. Use the initial sample code provided as a reference.
```

Comment:
It will be good to call out that Script mode automatically imports the required modules. When transitioning to Add-on mode, the developers will have to add those modules manually.